src/clockcache.c

```diff
@@ -1816,20 +1816,25 @@ clockcache_init(clockcache        *cc,   // OUT
    cc->heap_id = hid;
 
    /* lookup maps addrs to entries, entry contains the entries themselves */
-   cc->lookup =
-      TYPED_ARRAY_MALLOC(cc->heap_id, cc->lookup, allocator_page_capacity);
+   platform_memfrag memfrag_cc_lookup;
+   cc->lookup = TYPED_ARRAY_MALLOC_MF(
+      cc->heap_id, cc->lookup, allocator_page_capacity, &memfrag_cc_lookup);
```

Comment:
This is the 1st example of use of the MF-named array-allocator; where the `memfrag{} *` pointer is passed-in through the allocator macro.

This interface exists as the object for which the allocation is being done, i.e., `cc->lookup`, is not amenable to name translation using `##` preprocessor directives. 

[ See further below for explanations on other examples of new interface usages. ]